### PR TITLE
include harshad16 as ml developer experience sign chair

### PR DESCRIPTION
**Pull request by harshad16**
include harshad16 as ml developer experience sign chair

## Description
include harshad16 as ml developer experience sign chair
- i will try to bring in experience from IDE space, for ML experience from UI to Dev.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: sig-ml-developer-experience/OWNERS
Comment (andrewballantyne) [inline sig-ml-developer-experience/OWNERS:18]: Probably could just delete them -- they have not been active for a long while.

File: sig-ml-developer-experience/OWNERS
Comment (andrewballantyne) [inline sig-ml-developer-experience/OWNERS:6]: Best drop Erwan and Jay as well. I completely forgot about this file when I did my last pass.

File: sig-ml-developer-experience/OWNERS
Comment (harshad16) [reply sig-ml-developer-experience/OWNERS]: `emeritus_approvers` is for people who were approvers and no longer active.
would like me to remove them completely off the chart ?

Comment (andrewballantyne) [reply sig-ml-developer-experience/OWNERS]: Yeah there is no reason to keep such details at this stage. We are effectively starting fresh here.

**Review by andrewballantyne — APPROVED** [review]

**Comment by andrewballantyne** [comment]
@kywalker-rh need your approval here

/assign @kywalker-rh

**Review by kywalker-rh — APPROVED** [review]
Looks good

**Pull request merged**
